digiroad2-api-oth/src/test/scala/fi/liikennevirasto/digiroad2/MunicipalityApiSpec.scala

```diff
@@ -260,9 +264,9 @@ class MunicipalityApiSpec extends FunSuite with Matchers with BeforeAndAfter {
     when(mockRoadLinkService.getRoadLinksAndComplementariesFromVVH(Set(linkId), false)).thenReturn(Seq(newRoadLink))
     when(mockRoadLinkService.fetchVVHRoadlinkAndComplementary(linkId)).thenReturn(Some(newFetchedRoadLink))
     
-    val timeStamp = RoadLinkClient.createVVHTimeStamp()
-    when(mockRoadLinkClient.createVVHTimeStamp(any[Int])).thenReturn(timeStamp)
-    when(mockRoadLinkClient.createVVHTimeStamp(any[Int])).thenReturn(timeStamp)
+    val timeStamp = LinearAssetUtils.createTimeStamp()
+    when(mockRoadLinkClient.createTimeStamp(any[Int])).thenReturn(timeStamp)
+    when(mockRoadLinkClient.createTimeStamp(any[Int])).thenReturn(timeStamp)
```

Comment:
mihis tätä tarvitaan enään? Luodaanko jossakin timetstamp roadLinkClient kautta? Niissä tilanteissa luodaan suoraan LinearAssetUtils kautta. roadLinkClient ei kuuluisi omasta mielestä luoda mitään timestamp.

Reply:
poistettu nämä myös

Reply:
noi testit ei alkanut hajoamaan ?

Reply:
Ei, ajoin ton nyt vielä varmuuden vuoksi uudestaan. Nyt se timestamp tulee suoraan utilsista, niin ei tarvi mockata. Mock ei palauttanut tässä mitään ennalta asetettua kovakoodattua arvoa, vaan sen mitä funktio normaalisti palauttaa, niin mun nähdäkseni logiikan ei pitäis muuttua.

Reply:
joku copy paste koodaus hommaa ollut varmaan näissä